pyworkflow/constants.py

```diff
@@ -40,7 +40,7 @@
 VERSION_1_1 = '1.1.0'
 VERSION_1_2 = '1.2.0'
 VERSION_2_0 = '2.0.0'
-VERSION_3_0 = '3.0.4'
```

Comment:
I think for the future will be better to bump the version in the PR to master when finally attempting to make a release. ;)